charts/telemetry/0.1.0/values.yaml

```diff
@@ -35,11 +35,11 @@ ingress:
 
 resources:
   limits:
-   cpu: 300m
-   memory: 512Mi
+   cpu: 100m
+   memory: 128Mi
   requests:
-   cpu: 300m
-   memory: 512Mi
+   cpu: 100m
```

Comment:
where are these coming from ?

Reply:
They are mostly just made up, but they are based on the fact that cpu/mem usage in production seems to be pretty low.

Reply:
lets double the limit.

Reply:
Done, see latest commit.